core/src/main/java/nl/nn/adapterframework/doc/InfoBuilderSource.java

```diff
@@ -42,8 +42,8 @@
 import org.springframework.util.Assert;
 import org.xml.sax.SAXException;
 
-import nl.nn.adapterframework.doc.objects.IbisBean;
 import nl.nn.adapterframework.doc.objects.ChildIbisBeanMapping;
+import nl.nn.adapterframework.doc.objects.IbisBean;
```

Comment:
Hier verander je puur en import, dit vervuilt die git history en maakt git-blame onoverzichtelijker. Omdat er verder niets anders veranderd is in dit bestand, zou jij dit terug kunnen draaien?

Reply:
Dit is ontstaan door Ctrl-Shift-O in Eclipse. Als jullie deze omwisseling van imports nu accepteren, dan kan iedere developer na mij vrijelijk editen, Ctrl-Shift-O doen en edits ongedaan maken zonder dat dit probleem weer optreedt. Wat vind je daarvan?